src/Test.UnitTests.Sarif/Visitors/RemoveOptionalDataVisitorTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information. 
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Visitors
+{
+    public class RemoveOptionalDataVisitorTests
+    {
+        private SarifLog SampleLog = new SarifLog()
+        {
+            Runs = new List<Run>()
+            {
+                new Run()
+                {
+                    Results = new List<Result>()
+                    {
+                        new Result()
+                        {
+                            Guid = Guid.NewGuid().ToString(SarifConstants.GuidFormat)
+                        }
+                    }
+                }
+            }
+        };
+
+        [Fact]
+        public void RemoveGuids()
+        {
+            SarifLog log = SampleLog.DeepClone();
+
+            RemoveOptionalDataVisitor v = new RemoveOptionalDataVisitor(OptionallyEmittedData.None);
+            v.Visit(log);
+            Assert.NotNull(log.Runs[0].Results[0].Guid);
```

Comment:
>Assert [](start = 12, length = 6)

Code base convention: Use FluentAssertions:
```
log.Runs[0].Results[0].Guid.Should().NotBeNull();
```

Reply:
Oh -- I see that all of these baselining tests use the XUnit assertion library. Never mind (but do consider addressing this on a rainy day.)

---
In reply to: [390021991](https://github.com/microsoft/sarif-sdk/pull/1807#discussion_r390021991) [](ancestors = 390021991)

Reply:
I've fixed this file and others I added.